data/map.txt

```diff
@@ -27907,7 +27907,7 @@ planet "Kort Vek'kri"
 
 planet "Kraken Station"
 	attributes "near earth" station
-	landscape land/sivael7
+	landscape land/station8
```

Comment:
Kraken Station is described as being very old. sivael7 fits more of the old station vibe than station8.